### ISSUE TITLE
enhancement request in ticket creation

### ISSUE THREAD

**Issue by 1guardianofthegalaxy** [CrowdStrike/falcon-orchestrator#15]
When setting Detection Tab status to Remediation, then saving, a ticket is created.  Would like to have the Falcon portal Detect ID link in the ticket.  It appears in the email when Notification in the Ticketing tab is enabled and ticket is saved.  Would like to also have in the ticket.

**Comment by ivityc** [comment]
Yes this would be a required features or it would be difficult to trace back to a detection directly form an open ticket.
